axon-server-connector/src/main/java/org/axonframework/axonserver/connector/util/ExecutorServiceBuilder.java

```diff
@@ -78,4 +78,25 @@ static ExecutorServiceBuilder defaultQueryExecutorServiceBuilder() {
                 new AxonThreadFactory("QueryProcessor")
         );
     }
+
+
+    /**
+     * Create a default ExecutorServiceBuilder used to create a {@link ThreadPoolExecutor} for processing incoming
+     * query responses.
+     * Uses the {@link AxonServerConfiguration#getQueryResponseThreads()} as the core and maximum pool size, a
+     * keep-alive time of {@code 100ms}, the given {@link BlockingQueue} as the {@code workQueue} and an {@link
```

Comment:
For a thread pool that has min and max threads the same, the keep-alive time is not taken into account.

Reply:
I have adjusted the javadoc for this method, the other methods, and the builder methods. 
As well as removed the constant, since it is unused anyway.